policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml

```diff
@@ -1,10 +1,12 @@
 description: Indiana provides this base exemption amount.
 values:
-  2021-01-01: 1_000
+  2021-01-01: 1_000 #married filing jointly should be $2_000
```

Comment:
remove comment but do we account for this in the code

Reply:
We didn't account for this in the code but I modified this to add a breakdown by filing status